.github/workflows/playwright.yaml

```
@@ -117,6 +129,16 @@ jobs:
         env:
           JWKS_BASE64: ${{ secrets.JWKS_BASE64 }}
 
+      - name: Save Docker images to cache
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2162:info:2:78: read without -r will mangle backslashes

(shellcheck)